### ISSUE TITLE
work_horse_killed_handler Worker init argument not available in django-rq

### ISSUE THREAD

**Issue by gabriels1234** [rq/django-rq#579]
@selwin, Would you like me to add the option to provide a work_horse_killed_handler in django-rq (the same way the exception handlers are implemented)?

Thanks!

**Comment by selwin** [comment]
Yes, please! 😀On Mar 28, 2023, at 12:19 AM, gabriels1234 ***@***.***> wrote:﻿
@selwin, Would you like me to add the option to provide a work_horse_killed_handler in django-rq (the same way the exception handlers are implemented)?
Thanks!
